src/braze/tags/connectedContent.ts

```diff
@@ -93,6 +124,7 @@ export default <ITagImplOptions>{
 
     try {
       const jsonRes = JSON.parse(res.body)
+      console.log(jsonRes)
```

Comment:
there are a few `console.log`, are these intentional or for debugging purpose? If it's the latter please remove them.